Client/Frontend/UserContent/UserScripts/WalletEthereumProvider.js

```diff
@@ -3,13 +3,17 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-(function() {
+window.__firefox__.execute(function($) {
   
 if (window.isSecureContext) {
   function post(method, payload) {
-    return new Promise((resolve, reject) => {
-      webkit.messageHandlers.$<handler>.postMessage({
-        "securitytoken": "$<security_token>",
+    let postMessage = $(function(message) {
+      return $.postNativeMessage('$<handler>', message);
```

Comment:
> webkit.messageHandlers.walletEthereumProvider_1E024CC65FA245D7B5CD4DE80B6E15F8
< undefined

Should we use `$<handlerName>` here instead?

Reply:
Fixed. Used `$<message_handler>`